### PR TITLE
New religious sect - Children of the Kudzu (has nothing to do with kudzu)

### PR DESCRIPTION
**Pull request by Marmio64**
New religious sect based on crops: Children of the Kudzu.

Allows the chaplain to work more with his fellow service team friends: the hydroponics division. Improves RP.

Sacrifice:  Any grown crops work. The amount of favor you gain is based on how much nutriments the plant has. I tried to use potency, but that only works with seeds and having to put in ~4 seeds instead of 1 crop is incredibly tedious. Favor gained is 1+(nutriment x 2)

Rites
---------
Ent Conversion: Allows you to transform someone into a special wooden golem. What is special about them is they're a bit faster (same as gold golem) than standard golems, since wood golems by themselves aren't all that great. Costs 400 favor

------------
They don't get anything special for their bible blessing, just like the candle cult.

Oh also I fixed two spelling/grammar mistakes in the cult of st. Credit just because.

AND, this sect gets a green altar
kudzualtar

:cl:  
rscadd: Adds Children of the Kudzu sect
spellcheck: fixed a few typos in cult of st.credit 
/:cl:

### CODE REVIEW

**Comment by Nechronom** [comment]
looks neat! I like race changing rites. Also I thought the candle flame cult removed healing ability?

**Comment by Marmio64** [comment]
> looks neat! I like race changing rites. Also I thought the candle flame cult removed healing ability?

it does, its why i said its like the candle cult

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -319,6 +319,10 @@
 		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
 		return 1
 
+/datum/species/golem/wood/holy //slightly upgraded wood golem, for the plant sect
+	id = "holy wood golem"
+	speedmod = 1 //wood golems aren't very good, so the holy ones are slightly faster so that you don't put in a bunch of hardwork to downgrade yourself
+
```
Comment (ChesterTheCheesy) [inline code/modules/mob/living/carbon/human/species_types/golems.dm:327]: Add "changesource_flags = MIRROR_BADMIN" and "random_eligible = FALSE" so it's the same as other church golems, only available via the church conversion.

**Comment by ChesterTheCheesy** [comment]
I really don't like the strange seeds reward. Chaplain doesn't need them, botany does. Should this religion just make chaplain a better botanist? Also, 100 potency watermelons hold 20 nutriments, so you get 40 "favor points" from ONE melon, and you get 13 of them every plant yield. Meaning only one batch of top biomass generator allows you to make 15 strange seeds. Should i continue how this is not okay?

**Review by nesisknesis — APPROVED** [review]

**Comment by Marmio64** [comment]
> I really don't like the strange seeds reward. Chaplain doesn't need them, botany does. Should this religion just make chaplain a better botanist? Also, 100 potency watermelons hold 20 nutriments, so you get 40 "favor points" from ONE melon, and you get 13 of them every plant yield. Meaning only one batch of top biomass generator allows you to make 15 strange seeds. Should i continue how this is not okay?

They were more of something you could gift botany but i could find a replacement rite

**Comment by Cartlord** [comment]
> 
> 
> I really don't like the strange seeds reward. Chaplain doesn't need them, botany does. Should this religion just make chaplain a better botanist? Also, 100 potency watermelons hold 20 nutriments, so you get 40 "favor points" from ONE melon, and you get 13 of them every plant yield. Meaning only one batch of top biomass generator allows you to make 15 strange seeds. Should i continue how this is not okay?

Holy crap

**Comment by boodaliboo** [comment]
love new sects but i reccomend adding a second rite as most of them have 2 rites

**Comment by Marmio64** [comment]
> love new sects but i reccomend adding a second rite as most of them have 2 rites

if you've an idea for a second rite that is balanced, yeah ill add it, but i've no other ideas. Strange seeds are unbalanced, kudzu would be too antaggy, rites to give unstable mutagen/nutriment/etc would make the chaplain TOO self sufficient, and stuff like ambrosia gaia isn't even that great of a reward since botany will be rushing it anyhow. Also the technophile one only has a single rite too so it's not like having one rite is game breaking. Also, I was going to add a rite for podperson conversion but chances are the chap will go podperson anyhow to fit in with the theme, and podpeople aren't good enough to be costing you favor to convert into, especially when you basically have a upgraded version already (wood golems. Bit slower but space immune, stronger punches, natural armor, and they heal but don't die in darkness. Not having an exosuit slot is generally not too bad save for when you need fire resistance.)

**Comment by boodaliboo** [comment]
> > love new sects but i reccomend adding a second rite as most of them have 2 rites
> 
> if you've an idea for a second rite that is balanced, yeah ill add it, but i've no other ideas. Strange seeds are unbalanced, kudzu would be too antaggy, rites to give unstable mutagen/nutriment/etc would make the chaplain TOO self sufficient, and stuff like ambrosia gaia isn't even that great of a reward since botany will be rushing it anyhow. Also the technophile one only has a single rite too so it's not like having one rite is game breaking. Also, I was going to add a rite for podperson conversion but chances are the chap will go podperson anyhow to fit in with the theme, and podpeople aren't good enough to be costing you favor to convert into, especially when you basically have a upgraded version already (wood golems. Bit slower but space immune, stronger punches, natural armor, and they heal but don't die in darkness. Not having an exosuit slot is generally not too bad save for when you need fire resistance.)

heres a idea give a boost to passive healing if your a plant person and if your not then a incredibly shitty healing

**Comment by Marmio64** [comment]
> > > love new sects but i reccomend adding a second rite as most of them have 2 rites
> > 
> > 
> > if you've an idea for a second rite that is balanced, yeah ill add it, but i've no other ideas. Strange seeds are unbalanced, kudzu would be too antaggy, rites to give unstable mutagen/nutriment/etc would make the chaplain TOO self sufficient, and stuff like ambrosia gaia isn't even that great of a reward since botany will be rushing it anyhow. Also the technophile one only has a single rite too so it's not like having one rite is game breaking. Also, I was going to add a rite for podperson conversion but chances are the chap will go podperson anyhow to fit in with the theme, and podpeople aren't good enough to be costing you favor to convert into, especially when you basically have a upgraded version already (wood golems. Bit slower but space immune, stronger punches, natural armor, and they heal but don't die in darkness. Not having an exosuit slot is generally not too bad save for when you need fire resistance.)
> 
> heres a idea give a boost to passive healing if your a plant person and if your not then a incredibly shitty healing

Yeah but plantpeople already get passive healing and having it apply to nonplant people wouldnt be good

**Comment by boodaliboo** [comment]
> > > > love new sects but i reccomend adding a second rite as most of them have 2 rites
> > > 
> > > 
> > > if you've an idea for a second rite that is balanced, yeah ill add it, but i've no other ideas. Strange seeds are unbalanced, kudzu would be too antaggy, rites to give unstable mutagen/nutriment/etc would make the chaplain TOO self sufficient, and stuff like ambrosia gaia isn't even that great of a reward since botany will be rushing it anyhow. Also the technophile one only has a single rite too so it's not like having one rite is game breaking. Also, I was going to add a rite for podperson conversion but chances are the chap will go podperson anyhow to fit in with the theme, and podpeople aren't good enough to be costing you favor to convert into, especially when you basically have a upgraded version already (wood golems. Bit slower but space immune, stronger punches, natural armor, and they heal but don't die in darkness. Not having an exosuit slot is generally not too bad save for when you need fire resistance.)
> > 
> > 
> > heres a idea give a boost to passive healing if your a plant person and if your not then a incredibly shitty healing
> 
> Yeah but plantpeople already get passive healing and having it apply to nonplant people wouldnt be good

why not? just make it cost like 700 favor and make it shitty

**Comment by Marmio64** [comment]
> > > > > love new sects but i reccomend adding a second rite as most of them have 2 rites
> > > > 
> > > > 
> > > > if you've an idea for a second rite that is balanced, yeah ill add it, but i've no other ideas. Strange seeds are unbalanced, kudzu would be too antaggy, rites to give unstable mutagen/nutriment/etc would make the chaplain TOO self sufficient, and stuff like ambrosia gaia isn't even that great of a reward since botany will be rushing it anyhow. Also the technophile one only has a single rite too so it's not like having one rite is game breaking. Also, I was going to add a rite for podperson conversion but chances are the chap will go podperson anyhow to fit in with the theme, and podpeople aren't good enough to be costing you favor to convert into, especially when you basically have a upgraded version already (wood golems. Bit slower but space immune, stronger punches, natural armor, and they heal but don't die in darkness. Not having an exosuit slot is generally not too bad save for when you need fire resistance.)
> > > 
> > > 
> > > heres a idea give a boost to passive healing if your a plant person and if your not then a incredibly shitty healing
> > 
> > 
> > Yeah but plantpeople already get passive healing and having it apply to nonplant people wouldnt be good
> 
> why not? just make it cost like 700 favor and make it shitty

cause its a no downsides free passive healing that you can always trust and people can't take it away from you. Also it doesn't fit with a plant theme or anything, it'd be something more suitable for a different kind of sect, something healing/holy based.

**Pull request merged**